src/settings/draw_background.rs

```diff
@@ -0,0 +1,31 @@
+use glutin::{WindowedContext, PossiblyCurrent};
+use crate::{settings::SETTINGS, window::WindowSettings};
+
+#[cfg(target_os = "macos")]
+use glutin::platform::macos::WindowExtMacOS;
+use cocoa::appkit::{NSWindow, NSColor};
+use cocoa::base::{id, nil};
+use objc::{
+    rc::autoreleasepool,
+    runtime::{YES},
+};
+use csscolorparser::Color;
+
+#[cfg(target_os = "macos")]
+pub fn draw_background(window: &WindowedContext<PossiblyCurrent>) {
```

Comment:
For compartization's sake, I'm not sure if this window logic belongs into `src/settings`. It feels more like it should go into `src/window` instead.

Reply:
I moved it. Thank you !